accounts/api/tests.py

```diff
@@ -0,0 +1,233 @@
+# =================================================================================================
+#                                  All Rights Reserved.
+# =================================================================================================
+# File description:
+#           account api unit tests
+#
+# =================================================================================================
+#    Date      Name                    Description of Change
+# 12-Aug-2021  Wayne Shih              Initial create
+# $HISTORY$
+# =================================================================================================
+
+
+from django.contrib.auth.models import User
+from django.test import TestCase
+from rest_framework import status
+from rest_framework.test import APIClient
+
+
+LOGIN_URL = '/api/accounts/login/'
+LOGOUT_URL = '/api/accounts/logout/'
+SIGNUP_URL = '/api/accounts/signup/'
+LOGIN_STATUS_URL = '/api/accounts/login_status/'
+
+
+# <Wayne Shih> 12-Aug-2021
+# - tests.py could be under any directory in fact.
+#   as long as the test class is inherited from TestCase.
+# - Need to start with "test_" to name all unit tests as a method in the case.
+# - Each "test_" method is a unit test, which could include multiple cases
+#   in order to be efficient.
+# - Note: Django's unit tests take around 1 hour, which is relatively slow.
+# - Use
+#     $ python manage.py test
+#   or
+#     $ python manage.py test -v2
+#   to run tests.
+# - It creates a test database, e.g. 'test_twitter', everytime we run the test
+#   so that it will not mess up the original database 'twitter'.
+class AccountApiTests(TestCase):
```

Comment:
Do the test cases cover all of the codes?

Reply:
![image](https://user-images.githubusercontent.com/65328145/130333058-791061b1-2d23-4e21-ae5b-0d2760e4809e.png)